src/index.js

```diff
@@ -0,0 +1,5 @@
+import './style.css';
+
+function component() {}
+
+document.body.appendChild(component());
```

Comment:
Good job setting up webpack, however you have not completed the project requirements as per:

- There should be an array of dummy Todo lists
- There should be a function that iterates through the array to create the to-dos
- The arrays are displayed on screen when it leads

Kindly implement these description to make your submission in accordance with the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Create%20an%20index.js,values%20for%20each%20task.)